test/NuGet.Core.Tests/NuGet.ProjectModel.Test/PackageSpecOperationsTests.cs

```diff
@@ -11,6 +12,82 @@ namespace NuGet.ProjectModel.Test
 {
     public class PackageSpecOperationsTests
     {
+
+        [Fact]
+        public void AddOrUpdateDependency_AddsNewPackageDependencyToAllFrameworks()
```

Comment:
AllFrameworks isn't very fitting for a test with one framework :smile:

Reply:
This is where an upside down shuffle parrot is appropriate :D